lib/popup.html

```diff
@@ -5,22 +5,39 @@
      <link rel="stylesheet" href="../assets/popup.css">
   </head>
   <body>
-    <header>
-      <img src="../assets/white_logo.png" alt="Qowala logo"/>
-      <span>Hey, it's your friends at Qowala!</span>
-      <div></div>
-    </header>
-    <div class="non-measure-display">
-      <h3>This website is not tracked!</h3>
-    </div>
-    <div class="measure-display">
-      <h3>You've spent...</h3>
-      <div class="time-spent"></div>
-      <div class="parameters">
-        On <select class="parameters__dropdown network"></select>
-        <select class="parameters__dropdown date"></select>
+    <main id="app">
+      <header>
+        <img src="../assets/white_logo.png" alt="Qowala">
+        <hr>
+      </header>
+      <div v-if="tracked">
+        <p class="text secondary">
+          You've spent…
+        </p>
+        <div class="time-spent">
+          {{ timeSpent }}
+        </div>
+        <div class="parameters">
+          On
+          <select v-model="hostname" class="parameters__dropdown network">
+            <option v-if="noData" disabled :value="hostname">Please select another site</option>
```

Comment:
Yes it is good to be explicit and inform the user, but here I don't understand the use case. 
Looking at your code, the message `You didn't visited this site ${this.date.toLowerCase()}!` only displays when the website dropdown is on **"Please select another site"**. It seem it may not happen in any other case.

Trying the plugin locally, when you switch the date dropdown, by default it goes to **"Please select another site"**, so you have the message **"You didn't visited this site yesterday!"** which is displayed. But if you would remove this from the dropdown, you will never have the case that data is missing for a selected website, because it is impossible for you to select a website for which there is no data, as it is not displayed in the dropdown.
I really don't understand in which case this is useful ^^'

What is the case you encountered which made you add this logic?